### PR TITLE
Various fixes and improvements in documentation.

### PR DESCRIPTION
**Pull request by b-abel**
* Fixes in the documentation of Telemetry API and Event API
* Improve chapter pages 
* Harmonize formatting of remarks using "NB"

### CODE REVIEW

File: site/documentation/content/api/event/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The following operations can be used by *Protocol Adapters* to forward event mes
 **Preconditions**
 
 1. Adapter has established an AMQP connection with the AMQP Messaging Network.
-1. Adapter has established an AMQP link in role *sender* with the AMQP Messaging Network using target address `event/${tenant_id}` where `${tenant_id}` is the ID of the tenant that the client wants to upload event messages for. 
+1. Adapter has established an AMQP link in role *sender* with the AMQP Messaging Network using target address `event/${tenant_id}` where `${tenant_id}` is the ID of the client for which the client wants to forward event messages.
```
Comment (sophokles73) [inline site/documentation/content/api/event/index.md]: It was correct before FMPOV and is now changed for the worse as the tenant ID is definitely not the ID of the client.
Please revert this change.

File: site/documentation/content/api/telemetry/index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The following operations can be used by *Protocol Adapters* to forward telemetry
 **Preconditions**
 
 1. Adapter has established an AMQP connection with the AMQP Messaging Network.
-1. Adapter has established an AMQP link in role *sender* with the AMQP Messaging Network using target address `telemetry/${tenant_id}` where `${tenant_id}` is the ID of the tenant that the client wants to upload telemetry data for. 
+1. Adapter has established an AMQP link in role *sender* with the AMQP Messaging Network using target address `telemetry/${tenant_id}` where `${tenant_id}` is the ID of the client for which the client wants to forward telemetry data.
```
Comment (sophokles73) [inline site/documentation/content/api/telemetry/index.md]: see above

File: site/documentation/content/api/telemetry/index.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ All other combinations are not supported by Hono and may result in the terminati
 
 As indicated above, it is up to the discretion of the protocol adapter whether it wants to use *AT LEAST ONCE* or *AT MOST ONCE* delivery semantics.
 
-Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when uploading telemetry data.
+Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data.
```
Comment (sophokles73) [inline site/documentation/content/api/telemetry/index.md]: the devices indicate the delivery semantics when they upload the data. Please revert

File: site/documentation/content/api/telemetry/index.md [hunk 4]
```diff
@@ -39,9 +39,9 @@ All other combinations are not supported by Hono and may result in the terminati
...
 
-The following sequence diagram illustrates the flow of messages involved in the *HTTP Adapter* forwarding an *unsettled* telemetry data message to the downstream AMQP Messaging Network implementing *AT MOST ONCE* delivery semantics.
+The following sequence diagram illustrates the flow of messages involved in the *HTTP Adapter* forwarding an *settled* telemetry data message to the downstream AMQP Messaging Network implementing *AT MOST ONCE* delivery semantics.
```
Comment (sophokles73) [inline site/documentation/content/api/telemetry/index.md]: no, we are indeed always sending unsettled messages. The adapter simply doesn't wait for the outcome of the transfer before sending back a 202 to the device ... please revert

Comment (b-abel) [reply site/documentation/content/api/telemetry/index.md]: I had to read the passage in the documentation several times to understand what was meant. Maybe it should be made more explicit that the HTTP adapter implements both delivery semantics with *unsettled* mode. 

We could change the sentence right before this from:

> Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data.

to:

> Hono's HTTP adapter allows devices to indicate, which delivery semantics they want to use when forwarding telemetry data; the message is always forwarded as *unsettled*.

WDYT?

Comment (sophokles73) [reply site/documentation/content/api/telemetry/index.md]: > Hono's HTTP adapter allows devices to indicate which delivery semantics they want to use when uploading telemetry data. The HTTP adapter always forwards messages *unsettled* and either ignores the outcome of the message transfer (AT_MOST_ONCE) or waits for the downstream peer to accept the message (AT_LEAST_ONCE)  before acknowledging the reception of the message to device.

WDYT?

**Comment by b-abel** [comment]
Added the requested changes.
